### PR TITLE
Add Likely

### PR DESCRIPTION
**Pull request by ruLait**

### CODE REVIEW

File: files/likely/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Ilya Birman"
+github = "https://github.com/ilyabirman/Likely"
+homepage = "http://ilyabirman.net/projects/likely/"
+description = "Likely is the social sharing buttons that aren’t shabby."
+mainfile = "release/likely.js"
```
Comment (jimaek) [inline files/likely/info.ini]: Just `likely.js` here

**Comment by ruLait** [comment]
What's wrong? :\

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by ruLait** [comment]
Thanks!

**Pull request closed without merging**